tests/integration/test_spans.py

```diff
@@ -653,30 +653,33 @@ def test_span_ingestion(
             "org_id": 1,
             "project_id": 42,
             "retention_days": 90,
-            "tags": {
-                "span.op": "resource.script",
-                "span.group": "8a97a9e43588e2bd",
-            },
```

Comment:
The tag `span.group` no longer exists, is this intended?

Reply:
this diff is confusing, but the resource.script is moved further down with span.group - this is the span for db.query now